.teamcity/_self/projects/WPComTests.kt

```diff
@@ -298,6 +298,9 @@ fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): Bui
 
 					mkdir -p logs
 					find test/e2e -name '*.log' -print0 | xargs -r -0 tar cvfz logs.tgz
+
+					mkdir -p trace
+					find test/e2e/results -name '*.zip' -print0 | xargs -r -0 mv -t trace
```

Comment:
I had initially set up the necessary parts in https://github.com/Automattic/wp-calypso/pull/59143, but forgot to include this portion that actually collects the trace files.